plugin/color.py

```diff
@@ -2,12 +2,13 @@
 from .core.documents import is_transient_view
 from .core.protocol import Range
 from .core.protocol import Request
-from .core.registry import session_for_view, sessions_for_view, configurations_for_view
 from .core.registry import LSPViewEventListener
+from .core.registry import session_for_view, sessions_for_view, configurations_for_view
 from .core.settings import settings
 from .core.typing import List, Dict, Optional
 from .core.url import filename_to_uri
 from .core.views import range_to_region
+from .core.windows import debounced
```

Comment:
Not sure "windows.py" is the best place for it but don't have a better suggestion than a generic "utility".

Reply:
I wanted to put it in registry.py, but then I got a circular import problem. And having another source file just for a function is a bit overkill.